pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
**Suggestion:** Consider adding a comment to explain the purpose of the new settings added to the lockfile. This will help other developers understand why these settings were added.
```suggestion
settings:
  # autoInstallPeers: Automatically install peer dependencies
  autoInstallPeers: true
  # excludeLinksFromLockfile: Do not include linked packages in the lockfile
  excludeLinksFromLockfile: false
```